controller/articles.controller.js

```diff
@@ -60,8 +61,19 @@ function patchArticleById(req, res, next) {
   .catch((err) => {
     next(err)
   })
+}
+
 
+function deleteCommentById(req, res, next) {
+  const {comment_id} = req.params;
+  removeCommentById(comment_id).then((deletedComment) => {
+    res.status(204).send({ deletedComment }); 
```

Comment:
Express will not send back a response body with a 204 status. This should be either `res.sendStatus(204)` or `res.status(204).send()`